lib/tensorflow/graph.rb

```diff
@@ -86,18 +87,45 @@ def define_op(opName, name , input, device, attrs)
     node
   end
 
+  #
+  # Creates a constant Tensor that is added to the graph with a specified name.
+  # Official documentation of {tf.constant}[https://www.tensorflow.org/versions/r0.9/api_docs/python/constant_op.html#constant].
+  #
+  def constant(name, data, type)
```

Comment:
@Arafatk shouldn't we try and follow as much as possible the same interface (order of arguments) as the Python wrapper? I.e. `tf.constant(value, dtype=None, shape=None, name='Const')`

https://www.tensorflow.org/versions/r0.9/api_docs/python/constant_op.html#constant